MAINTAINERS.md

```diff
@@ -1,3 +1,26 @@
+## Specification maintainers
+
+### SAM/BAM
+
+* James Bonfield (@jkbonfield)
+* Yossi Farjoun (@yfarjoun)
+
+### CRAM
+
+* James Bonfield (@jkbonfield)
+* Vadim Zalunin (@vadimzalunin)
```

Comment:
I believe that @vadimzalunin will be departing from his role as CRAM maintainer in the near future. @vadimzalunin Do you have a replacement lined up?